user/precise-to-trusty-migration-guide.md

```diff
@@ -5,4 +5,17 @@ layout: en
 
 Ubuntu Trusty is becoming the default Linux distribution instead of Ubuntu Precise on Travis CI. Here are the most common issues our customers ran into and how you can fix them.
 
-> If you’d like to stay on Ubuntu Precise or need more time to set up your repository with Ubuntu Trusty, please explicitly set `dist: precise` in your .travis.yml file as soon as possible.
+# Staying on Precise
+
+If you’d like to stay on Ubuntu Precise or need more time to set up your repository with Ubuntu Trusty, please explicitly set `dist: precise` in your .travis.yml file as soon as possible.
+
+# PyPy Support
+
+Using `pypy2` and `pypy3` for selecting PyPy versions is not supported anymore, instead you need to specify the full version number, e.g.: `python: pypy2.7-5.8.0`.
```

Comment:
Just a side note (since `pypy` and `pypy3` should work now): I do not think `pypy2` was ever supported.